src/assets/spinner.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="110" height="110" viewBox="0 0 110 110" fill="none" xmlns="http://www.w3.org/2000/svg">
+<circle cx="57.5" cy="102.5" r="7.5" fill="white"/>
+<circle cx="37" cy="101" r="6" fill="white"/>
+<circle cx="96" cy="43" r="12" fill="white"/>
+<circle cx="95" cy="73" r="11" fill="white"/>
+<circle cx="79" cy="94" r="9" fill="white"/>
+</svg>
```

Comment:
no indentation!

Reply:
Consider using svgo to optimize svg file :)

https://github.com/svg/svgo

Reply:
you may want to consider grouping the filling color:
```
<svg width="110" height="110" viewBox="0 0 110 110" fill="none" xmlns="http://www.w3.org/2000/svg">
  <g fill="white">
    <circle cx="57.5" cy="102.5" r="7.5"/>
    <circle cx="37" cy="101" r="6"/>
    <circle cx="96" cy="43" r="12"/>
    <circle cx="95" cy="73" r="11"/>
    <circle cx="79" cy="94" r="9"/>
  </g>
</svg>
```
Furthermore, assuming you will be `styled-componenting` svg, you can further make the SVG being more React-like, for example with fill color being provided via css
```
const Svg = styled.svg`
 color: rgba(255, 255, 255, 1);
`
// then, in the svg alone:
fill="currentColor"
```

Reply:
@dotadota should I optimize this 7 simple lines of svg? I don't see any prop which is unnecessary in this file. Please tell me why I need it now